daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/SchemaSet.scala

```diff
@@ -356,7 +356,15 @@ final class SchemaSet private (
         ge
       }
       case RootSpec(None, rootElementName) => {
-        findRootElement(rootElementName)
+        val possibleRoots: Seq[GlobalElementDecl] = schemaSet.schemas.flatMap {
+          _.schemaDocuments.flatMap(_.searchGlobalElementDecl(rootElementName))
+        }
+        if (possibleRoots.length == 1) {
+          possibleRoots.head
+        } else {
+          val qn = RefQName(None, rootElementName, NoNamespace)
```

Comment:
Add some comments explaining the rationale here. E.g., 
```
It's ambiguous what root element, and no namespace URI was provided. 
So we assume not specifying a namespace URI may mean the intention 
was to choose the root that it is in No Namespace. 
If there is no such, then the diagnostic will encourage user to specify
the namespace explicitly. 
```

Reply:
> So we assume not specifying a namespace URI may mean the intention

I'm not sure that's a safe assumption. It's possible the user just didn't realize there were two elements with the same name and they actually did want the namespaced one. Feels like it would be safer to not assume and instead error saying there's an ambiguity and just require that the user be explicit about which namespace they want. It believe it's always possible to set no-namespace if that is needed for example one can use `{}foo` in the CLI. Is there a way to do it in the TDML Runner? Maybe `rootNS=""` is explicitly no-namespace?